src/__tests__/actions/resettingPassword.action.test.js

```diff
@@ -0,0 +1,47 @@
+import expect from 'expect';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import resetPasswordRequest from '../../actions/resetingPassword.action';
+import CONSTANTS from '../../constants/index';
+import RESPONSES from '../../mock/responses';
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+let store;
+const { RESETTING_ERROR_ACTION, RESETTING_PASSWORD_ACTION } = CONSTANTS;
+
+describe('Password reset Action tests', () => {
+  const mock = new MockAdapter(axios);
+  const token = 'xaAaddddAD3323@#@$$@%@';
+  const password = 'AverysecurePassword';
+  const url = `${process.env.REACT_APP_BASE_URL}/users/password_reset?token=${token}`;
```

Comment:
You don't need to attach the base url here. Just have it as `const url = `users/password_reset?token=${token}`;`